pkg/overlay/cache.go

```diff
@@ -132,6 +130,10 @@ func (node *NodeDossier) Online() bool {
 		node.Reputation.LastContactSuccess.After(node.Reputation.LastContactFailure)
 }
 
+func (node *NodeDossier) Valid(maxStats *NodeStats) bool {
```

Comment:
Should we call nodes that meet the uptime and audit success criteria as "vetted" instead of "valid"?

Reply:
that seems better than valid, given the whitepaper's terminology

Reply:
_vetted_ sound better to me for the same reason than @wthorp mentioned